adapters/AppLovin/AppLovinAdapter/RTB/AppLovinRtbAdapter.m

```diff
@@ -0,0 +1,137 @@
+//
+//  AppLovinRtbAdapter.m
+//  Adapter
+//
+//  Created by Christopher Cong on 7/17/18.
+//  Copyright © 2018 Google. All rights reserved.
+//
+
+#import "AppLovinRtbAdapter.h"
+#import "GADMAdapterAppLovinExtras.h"
+#import "GADMAdapterAppLovinUtils.h"
+#import "GADMAdapterAppLovinConstant.h"
+#import "GADMAppLovinRtbBannerRenderer.h"
+#import "GADMAppLovinRtbInterstitialRenderer.h"
+#import "GADMAppLovinRtbRewardedRenderer.h"
+
+#import <AppLovinSDK/AppLovinSDK.h>
+
+@interface AppLovinRtbAdapter ()
+// Controlled Properties
+@property (nonatomic, copy) GADMediationServerConfiguration *configuration;
+@property (nonatomic, copy) GADRTBSignalCompletionHandler signalCompletionHandler;
```

Comment:
Also, what's the point of these two properties?

Reply:
First isn't needed anymore. Second one, AdMob wants us to retain it.